src/components/modals/AddAccounts/steps/04-step-finish.js

```diff
@@ -50,13 +50,10 @@ function StepFinish({ currency, t, checkedAccountsIds }: StepProps) {
 
 export default StepFinish
 
-export const StepFinishFooter = ({ onCloseModal, onGoStep1, t }: StepProps) => (
+export const StepFinishFooter = ({ onGoStep1, t }: StepProps) => (
   <Fragment>
-    <Button mr={2} outline onClick={onGoStep1}>
+    <Button mr={2} primary onClick={onGoStep1}>
       {t('app:addAccounts.cta.addMore')}
     </Button>
-    <Button primary onClick={onCloseModal}>
-      {t('app:common.close')}
-    </Button>
```

Comment:
why is is gone?

Reply:
@bscholving ok with this?

Reply:
@khalilbenihoud requested for it

Reply:
would be great if it was tracked/written somewhere, we have github we may use it

Reply:
yes, because we already have a cross ot close it and you can close it by clicking outside, it is like 3rd time and it was the main focus - primary button

Reply:
ok . make sense 👌